test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -30,6 +30,12 @@ Describe "Import-Module" -Tags "CI" {
         (Get-Module -Name $moduleName).Name | Should -BeExactly $moduleName
     }
 
+    It "should be able to load a module with a trailing directory separator" {
+        $modulePath = (Get-Module -ListAvailable $moduleName).ModuleBase + [System.IO.Path]::DirectorySeparatorChar
+        Import-Module -Name $modulePath
+        (Get-Module -Name $moduleName).Name | Should -BeExactly $moduleName
```

Comment:
The module already was loaded in previous test.
Seems we could check:
```powershell
{ Import-Module -Name $modulePath -ErrorAction Stop } | Should -Not -Throw
``

Reply:
And maybe add `-Force` to really load the module and test whole code.

Reply:
Perhaps a `Remove-Module` would make more sense here